docs/tutorial.rst

```diff
@@ -81,3 +81,24 @@ including an introduction to the module, and how to utilize the seeding feature.
 
    tutorials/matching/faq
    tutorials/matching/sgm
+   tutorials/matching/padded_gm
+
+.. _subgraph_tutorials:
+
+Subgraph
+========
+The following tutorial demonstrates how to estimate the signal-subgraph of samples of a graph/class model according to either the coherent or incoherent estimator models.
+
+.. toctree::
+   :maxdepth: 1
+
+   tutorials/subgraph/subgraph
+
+Aligning
+========
+The following tutorials shows how to align two seperate datasets with each other.
+
+.. toctree::
+   :maxdepth: 1
+
+   tutorials/matching/aligning
```

Comment:
aligning != matching; shouldn't go inside.

Reply:
as in shouldn't go inside of the matching folder

Reply:
or is this a misprint>

Reply:
That's a misprint, sorry.